.github/workflows/test.yml

```diff
@@ -84,4 +84,26 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: Lint Javascript + SCSS 👕
-        run: npm run lint
\ No newline at end of file
+        run: npm run lint
+
+  typecheck:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install Dependencies ⬆️
+        run: |
+          npm ci
+          npx lerna bootstrap
+      - name: Run TypeScript type checking ✅
```

Comment:
Babel just removes the TypeScript types, and doesn't do any actual type checking. Because of that, we now need an explicit type-checking step.